pkgs/development/python-modules/py-partiql-parser/default.nix

```
@@ -18,10 +19,11 @@ buildPythonPackage rec {
     owner = "getmoto";
     repo = "py-partiql-parser";
     rev = "refs/tags/${version}";
-    hash = "sha256-BPap4f9ro269K50qpVlTiEPNyyMCvBHCr2obuSfuNos=";
+    hash = "sha256-BSqc3xibStb3J6Rua4dDp/eRD5/ns/dU1vGa4vL1Cyo=";
   };
 
   nativeBuildInputs = [
```

```suggestion
  build-system = [
```
There is a new parameter, `build-system`, for build-time only Python dependencies.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

---

Applies to `moto` as well.